http_cache.rst

```diff
@@ -192,9 +192,8 @@ These four headers are used to help cache your responses via *two* different mod
 
     All of the HTTP headers you'll read about are *not* invented by Symfony! They're
     part of an HTTP specification that's used by sites all over the web. To dig deeper
-    into HTTP Caching, check out the original `RFC 2616`_ document, or these two
```

Comment:
@xabbuh, why do you think it makes sense? 
Mark Nottingham recommends to forget about 2616 and use newer specs:
'These documents are now RFCs that collectively replace RFC2616, and should be [referenced and used in its place] (https://www.mnot.net/blog/2014/06/07/rfc2616_is_dead)'.